k8s/README.md

```diff
@@ -202,6 +202,36 @@ To send a request to the API
     curl -d '{"message":"hello world"}' -H "content-type:application/json" ${MINIKUBE_IP}:${NODE_PORT}/echo?key=${ENDPOINTS_KEY}
     ```
 
+## Using GCE L7 Load Balancer with ESP
+
+[GLBC](https://github.com/kubernetes/contrib/tree/master/ingress/controllers/gce) is a Kubernetes ingress controller
+that configures external loadbalancers. We can use GLBC to route traffic from an external IP address to an ESP-enabled
+Kubernetes service. By default, [Google Container Engine](https://cloud.google.com/container-engine/) deploys GLBC as a
+cluster addon. If you do not use Google Container Engine, then you need to
+[install GLBC](https://github.com/kubernetes/contrib/tree/master/ingress/controllers/gce) in your cluster.
+
+To use GLBC with ESP, we need to create an ingress resource in addition to a service and a deployment.
+Edit the file `esp_echo_gke_ingress.yaml` and replace service name and config version with your values.
+Notice that we use `-z` flag and a `readinessProbe` to define a health-checking endpoint on the application
+port. Deploy these resources to your cluster:
+
+
+  ```
+  kubectl create -f esp_echo_gke_ingress.yaml
+  ```
+
+You can check the status of your ingress using the following command:
+
+  ```
+  kubectl describe ingress
+  ```
+
+Once you obtain an external address, we can send a request to the echo backend:
+
+  ```
+  curl -d '{"message":"hello world"}' -H "content-type:application/json" http://${INGRESS_IP_ADDRESS}/echo?key=${ENDPOINTS_KEY}
```

Comment:
I see. Didn't notice it's mentioned above.

It's okay to use ENDPOINTS_KEY in your PR since it's not newly introduced. But we probably should change it to API_KEY. I never see the usage of ENDPOINTS_KEY anywhere else. I'll do a PR after yours if you agree.